src/Graphics/Effect/EffectParameter.cs

```diff
@@ -43,6 +43,12 @@ public int ColumnCount
 			private set;
 		}
 
+		public int ElementCount
```

Comment:
We'll have to make this internal, since I don't think it's in the public API.

Reply:
Yeah I'm fine with making it internal. We could also make it EXT, right?

Reply:
Let's make it internal, since that way we can make it a plain variable without the getter/setter in the way.